webapp/src/Controller/API/AbstractRestController.php

```diff
@@ -105,12 +106,20 @@ protected function performSingleAction(Request $request, string $id)
 
     /**
      * Render the given data using the correct groups
-     * @param Request $request
-     * @param mixed $data
+     *
+     * @param Request  $request
```

Comment:
Do we need the `@param` annotation here with the type hints below?

Reply:
Nope will remove them